index.html

```diff
@@ -1,29 +1,136 @@
 <!DOCTYPE html>
 <html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
+<head> 
+   <!--This section creates the title of the HTML document and links it the CSS file -->
+    <meta charset = "utf-8">
+    <title> Pepper Pete's Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+<body>  <!--This section outlines the header logo, title and header list -->
+   <header> <div class="head"> 
+      <img width="80" src="logo_v3.png" alt="header logo">
+      <h1 style="color:aliceblue; background-color: #E0643C; margin-right: auto"> PEPPER PETE'S</h1> 
+   
+   <!--these are the menu options -->
```

Comment:
This heading should actually be an `h2` 

And ideally I would have preferred to see these styles in your CSS file as opposed to inline